packages/smooth_app/lib/data_models/user_preferences_model.dart

```diff
@@ -159,13 +161,17 @@ class PreferencesVariable {
   static const String NOVA_GROUP = 'nova';
   static const String NUTRI_SCORE = 'nutriscore';
 
+  @deprecated
   static List<String> getMandatoryVariables() => <String>[
+        // TODO(monsieurtanuki): still relevant?
         VEGAN,
         VEGETARIAN,
         //GLUTEN_FREE,
       ];
 
+  @deprecated
   static List<String> getAccountableVariables() => <String>[
+        // TODO(monsieurtanuki): still relevant?
```

Comment:
not relevant, there's no notion of mandatory variables and accountable variables anymore.